startup/40-xspress3.py

```diff
@@ -211,12 +211,17 @@ def collect(self):
                           f'does not match the length of "self._datum_ids" ({len_datum_ids})')
 
         num_frames = min(len_di_timestamps, len_datum_ids)
+
+        # For the same timestamp for all frames, uncomment the next line, and comment the line within the loop.
+        # ts = float(ttime.time())
         for frame_num in range(num_frames):
             datum_id = self._datum_ids[frame_num]
             ts = di_timestamps[frame_num]
 
             data = {self.name: datum_id}
             # TODO: fix the lost precision as pymongo complained about np.float128.
+
+            # Comment the following line to use the same timestamp for frames.
```

Comment:
FYI, this commented line and the commented lines in green above were added during my last visit at QAS to resolve the issue #15.